SwiftScripts/Sources/TargetConfig/Target.swift

```diff
@@ -3,48 +3,49 @@ import ApolloCodegenLib
 
 public enum Target: CaseIterable {
   case starWars
-  case gitHub
+//  case gitHub
```

Comment:
In order to get generation of all targets to work, I commented this one out for now, since we haven't set it up yet and it would cause the whole script to fail.